src/android/Chromecast.java

```
@@ -566,6 +566,52 @@ public boolean mediaStop(CallbackContext callbackContext) {
 		return true;
 	}
 
+
+	/**
+	 * Handle Track changes.
+	 * @param activeTrackIds track Ids to set.
+	 * @param textTrackStyle text track style to set.
+	 * @param callbackContext
+	 * @return
+	 */
+	public boolean mediaEditTracksInfo(JSONArray activeTrackIds, JSONObject textTrackStyle, final CallbackContext callbackContext) {
+		long[] trackIds = new long[activeTrackIds.length()];
+
+		try {
+			for (int i = 0; i < activeTrackIds.length(); i++) {
+				trackIds[i] = activeTrackIds.getLong(i);
+			}
+		} catch (JSONException ignored) {
+

```

There are a few empty catch blocks, I would at least add an error message to alert developers.

---

Fixed